internal/pkg/describe/env.go

```diff
@@ -50,23 +65,68 @@ func NewEnvDescriber(projectName string, envName string) (*EnvDescriber, error)
 		return nil, fmt.Errorf("assuming role for environment %s: %w", env.ManagerRoleARN, err)
 	}
 	return &EnvDescriber{
-		env:          env,
-		store:        svc,
-		apps:         apps,
-		sessProvider: sess,
+		env:      env,
+		store:    svc,
+		proj:     proj,
+		apps:     apps,
+		rgClient: resourcegroups.New(sess),
 	}, nil
 }
 
 // Describe returns info about a project's environment.
 func (e *EnvDescriber) Describe() (*EnvDescription, error) {
-	var tags map[string]string
+	appsForEnv, err := e.filterAppsForEnv()
+	if err != nil {
+		return nil, err
+	}
+
 	return &EnvDescription{
 		Environment:  e.env,
-		Applications: e.apps,
-		Tags:         tags,
+		Applications: appsForEnv,
+		Tags:         e.proj.Tags,
 	}, nil
 }
 
+func (e *EnvDescriber) filterAppsForEnv() ([]*archer.Application, error) {
+	var appObjects []*archer.Application
+
+	tags := map[string]string{
+		stack.EnvTagKey: e.env.Name,
+	}
+	arns, err := e.rgClient.GetResourcesByTags(cloudformationResourceType, tags)
+	if err != nil {
+		return nil, fmt.Errorf("get resources for env %s: %w", e.env.Name, err)
+	}
+
+	stacksOfEnvironment := make(map[string]bool)
+	for _, arn := range arns {
+		stack, err := e.getStackName(arn)
+		if err != nil {
+			return nil, fmt.Errorf("get stack name from ARN %s: %w", arn, err)
```

Comment:
```suggestion
			return nil, err
```
We don't need to add extra context if the error is returned from a private method (the info is redundant). Instead, usually the only time we want to wrap errors is if it's returned from a public method
https://www.efekarakus.com/golang/2019/09/26/when-to-wrap-errors.html

Reply:
Ahh, that makes sense. Thanks. Will adjust tests accordingly, too.